app/controllers/stafftools/resources_controller.rb

```diff
@@ -2,6 +2,25 @@
 
 module Stafftools
   class ResourcesController < StafftoolsController
+    class StafftoolsRequest < Chewy::Search::Request
```

Comment:
The naming for `StafftoolsRequest` is unclear to me. Is there a better name we could use?

Reply:
`StafftoolsSearchRequest` Would be more clear or you could use `MultiIndexSearchRequest`.